schemas/v3.1/meta/oas.json

```diff
@@ -0,0 +1,1424 @@
+{
+  "$schema": "http://json-schema.org/draft/2019-WIP/schema#",
+  "$id": "https://spec.openapis.org/oas/3.1/meta/oas",
+  "$vocabulary": {
+    "https://specs.openapis.org/oas/3.1/vocab/oas": true
+  },
+  "$recursiveAnchor": true,
+  "title": "OpenAPI Schema Object Vocabulary Schema",
+  "type": ["object", "boolean"],
+  "properties": {
+    "nullable": {
+      "type": "boolean",
+      "default": false,
+      "deprecated": true
+    },
+    "discriminator": {
+      "$ref": "#/$defs/Discriminator"
+    },
+    "example": {
+      "deprecated": true
+    },
+    "externalDocs": {
+      "$ref": "#/$defs/ExternalDocumentation"
+    },
+    "xml": {
+      "$ref": "#/$defs/XML"
+    }
+  },
+  "patternProperties": {
+    "^x-": {
+    }
+  },
+  "unevaluatedProperties": false,
+  "$defs": {
+    "Discriminator": {
+      "type": "object",
+      "required": [
+        "propertyName"
+      ],
+      "properties": {
+        "propertyName": {
+          "type": "string"
+        },
+        "mapping": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "XML": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "namespace": {
+          "type": "string",
+          "format": "uri"
+        },
+        "prefix": {
+          "type": "string"
+        },
+        "attribute": {
+          "type": "boolean",
+          "default": false
+        },
+        "wrapped": {
+          "type": "boolean",
+          "default": false
+        }
+      },
+      "patternProperties": {
+        "^x-": {
+        }
+      },
+      "unevaluatedProperties": false
+    }
+  },
+  "Reference": {
+    "type": "object",
+    "required": [
+      "$ref"
+    ],
+    "patternProperties": {
+      "^\\$ref$": {
+        "type": "string",
+        "format": "uri-reference"
+      }
+    }
+  },
+  "Info": {
+    "type": "object",
+    "required": [
+      "title",
+      "version"
+    ],
+    "properties": {
+      "title": {
+        "type": "string"
+      },
+      "description": {
+        "type": "string"
+      },
+      "termsOfService": {
+        "type": "string",
+        "format": "uri-reference"
+      },
+      "contact": {
+        "$ref": "#/$defs/Contact"
+      },
+      "license": {
+        "$ref": "#/$defs/License"
+      },
+      "version": {
+        "type": "string"
+      }
+    },
+    "patternProperties": {
+      "^x-": {
+      }
+    },
+    "additionalProperties": false
+  },
+  "Contact": {
```

Comment:
These definitions are nothing to do with the schema object, which is what this (meta?)-schema file is describing in "properties", should I make another file which is just defs?